.github/workflows/publish.yml

```diff
@@ -57,6 +55,24 @@ jobs:
         env:
           BAIPP_DIST: ${{ steps.baipp.outputs.dist }}
 
+      - name: "Get PST version"
```

Comment:
I added these steps to the `prerelease` workflow but should have added them to the `publish` one too for some final quality checks on the packaged artefacts